ovos_plugin_common_play/ocp/player.py

```diff
@@ -254,7 +259,10 @@ def set_now_playing(self, track):
             if len(self.playlist):
                 self.now_playing.update(self.playlist[0])
             else:
+                # TODO This is a track with no URI; why should we try to play it
```

Comment:
this is just updating the metadata of the track, you can change it independently from playback. often when playback starts you may get extra info from the player itself such as duration etc

if some skill is handling playback natively there would be no track at all but OCP would keep track of the state of things